charts/fission-all/templates/deployment.yaml

```diff
@@ -680,7 +680,50 @@ spec:
           value: {{ .Values.traceSamplingRate | default "0.5" | quote }}
         - name: DEBUG_ENV
           value: {{ .Values.debugEnv | quote }}
+        # TLS authentication is TLS with authentication (2 way)
+        # More info: https://docs.confluent.io/current/kafka/authentication_ssl.html#ssl-overview
+        {{- if .Values.kafka.authentication.tls.enabled}}
```

Comment:
nit: Leave empty before `}}`  L685, L693, L695, L700

Reply:
Oops! Thank you for noticing it. Will fix it. Thank you!

Reply:
Fixed